server/src/main/java/org/opensearch/search/aggregations/metrics/AbstractInternalHDRPercentiles.java

```diff
@@ -144,8 +144,10 @@ public boolean keyed() {
 
     @Override
     public AbstractInternalHDRPercentiles reduce(List<InternalAggregation> aggregations, ReduceContext reduceContext) {
+        checkCancelled(reduceContext);
```

Comment:
Again, this check is redundant. Seeing this at few other places as well, which we can probably get rid of